README.MD

```diff
@@ -74,6 +74,74 @@ The loaders accepts the name of a loader that should be applied in advance. E.g.
 }
 ```
 
+### Custom HTML Translation extractors
+
+The htmlLoader supports registering custom HTML text extractors and rewrites the AngularJS extractor to be based on the same API.
+
+The API of an extractor is:
+
+```
+export interface HtmlTranslationExtractor {
+  (element: AngularElement, context: HtmlTranslationExtractionContext): void;
+}
+
+export interface AngularElement {
+  tagName: string;
+  attributes: Attribute[];
+  texts: Text[];
+  startPosition: number;
+}
+
+export interface HtmlTranslationExtractionContext {
+  emitError(message: string, position: number): void;
+  emitSuppressableError(message: string, position: number): void;
+  registerTranslation(translation: {
+    translationId: string;
+    defaultText?: string;
+    position: number;
+  }): void;
+
+  asHtml(): void;
+}
+```
+
+The extractor receives an angular element with all its attributes and its direct text siblings as well as a context that can be used to either register a new translation or emit a warning/error.
+The [translate-directive-translation-extractor.ts](src/html/translate-directive-translation-extractor.ts) contains an implementation of an extractor.
+Custom extractors can be specified with the html loader:
+
+```js
+{
+  loader: WebPackAngularTranslate.htmlLoader({
+    translationExtractors: [(element, context) => { ... }]
+  })
+}
+```
+
+WebpackAngularTranslates provides the `angularI18nTranslationsExtractor` to support extractions of translations in hybrid applications 
```

Comment:
I suggest adding a new section here for `angularI18nTranslationsExtractor`

Reply:
Plus is this extractor only for hybrid applications, or it can be used for angular only also?